gf-webapp-admin/src/component/rooms/room/slider/Slider.tsx

```diff
@@ -0,0 +1,24 @@
+import * as React from "react";
+import {IRoom} from "../../../../model/room";
+import {Slide} from "./slide/Slide";
+import styles from "./slider.module.scss";
+
+export const Slider = ({room}:{room:IRoom}) => {
+  const slider = Object.keys(room).filter(key => key.includes("slider"));
```

Comment:
What happens here? Slider better use with an array instead of filtration object by keys naming.

Reply:
Added a comment describing the problem